server/pages/index.jade

```diff
@@ -142,5 +142,20 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				script(src=urls[ chunk + '-min' ])
 		script(type='text/javascript')!='window.AppBoot();'
 
+		script.
+			if (window.console) {
+
+				if(true) {
+                    //Browser supports color output
+                    console.log("%cWait! This area is meant for developers!", "color: red;font-size: xx-large");
+				}else{
+					//No color supported
+                    console.log("Wait! This area is meant for developers!");
+				}
+                console.log("If someone told you to type or paste something into this area it could compromise the security of your account.");
+
+            }
```

Comment:
@dmsnell Yeah I'll clean that up. Also I was testing console on safari and realized that it and IE do not support 'rich' console output. Can I just do agent sniffing and only output plain text to log?